build_tools/buildkite/pipelines/benchmark.yml

```diff
@@ -0,0 +1 @@
+fragment/bootstrap-trusted.yml
```

Comment:
This symlink isn't necessary. The presubmit and postsubmit symlinks were only necessary because I reshuffled the directory structures and the pipelines already existed.